modules/veteran_confirmation/VETERAN_CONFIRMATION.yml

```diff
@@ -15,12 +15,15 @@ info:
 
     ### 2. Test the API
     In the endpoint documentation below, we've provided a curl command builder for trying out the API before implementation with your app.
+    Use [Test User](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts.md) attributes to populate the curl.
 
     ### 3. Build your app
     The base URI for the Veteran Confirmation API in the Development environment is:
     
     https://dev-api.va.gov/services/veteran_confirmation/v0
 
+    In this environment, use attributes from the list of [Test Users](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts.md). Only Test Users can return a `"confirmed"` response.
```

Comment:
Do you think it's worth mentioning that some of them may not return confirmed? I don't really know what the proportion is, so maybe that's not necessary at all!

Reply:
I think a larger recognition of that is probably necessary but am going to ignore it for this update.